src/vivarium_cluster_tools/psimulate/worker/load_test_work_horse.py

```diff
@@ -0,0 +1,60 @@
+"""
+================
+Load Test Worker
+================
+
+RQ worker executable for doing load testing.
+
+"""
+import time
+from traceback import format_exc
+
+import numpy as np
+import pandas as pd
+from loguru import logger
+from rq import get_current_job
+from vivarium.framework.randomness import get_hash
+
+from vivarium_cluster_tools.psimulate.environment import ENV_VARIABLES
+from vivarium_cluster_tools.psimulate.jobs import JobParameters
+
+LOAD_TEST_WORK_HORSE_IMPORT_PATH = f"{__name__}.work_horse"
+
+
+def work_horse(job_parameters: dict) -> pd.DataFrame:
+    node = f"{ENV_VARIABLES.CLUSTER_NAME.value}:{ENV_VARIABLES.HOSTNAME.value}"
+    job = f"{ENV_VARIABLES.JOB_NAME.value}: {ENV_VARIABLES.JOB_ID.value}:{ENV_VARIABLES.TASK_ID.value}"
+
+    job_parameters = JobParameters(**job_parameters)
+
+    test_type = job_parameters.extras["test_type"]
+    test_runner = {
+        "sleep": sleep_test,
+    }[test_type]
+
+    logger.info(f"Launching new job {job} on {node}")
+    logger.info(f"Starting job: {job_parameters}")
+    try:
+        return test_runner(job_parameters)
+    except Exception:
+        logger.exception("Unhandled exception in worker")
+        job = get_current_job()
+        job.meta["root_exception"] = format_exc()
+        job.save_meta()
+        raise
+    finally:
+        logger.info(f"Exiting job: {job_parameters}")
+
+
+def sleep_test(job_parameters: JobParameters) -> pd.DataFrame:
```

Comment:
So this is #1 in the ticket's listed test types, and we'll open other tickets for some of these other examples?

```
1. Start a job, sleep for passed in # seconds (as a parameter); Return an empty data frame 
2 .Starts and generates a numpyarray (to consume memory); Returns an empty data frame if successful
3. Starts and generates a data frame with large qty of long column names and returns them (something that looks like multiple myeloma) - "SwampMonster"
Others
```